food-truck-api/src/main/java/food/truck/api/search/TruckSearchService.java

```diff
@@ -36,7 +31,7 @@
 
         Query truckQuery = qb.keyword()
                 .fuzzy().withEditDistanceUpTo(2)
-                .onFields("name", "text_menu", "description", "foodCategory")
+                .onFields("name", "description") // TODO: Search in tags, as well
```

Comment:
Would adding searching in tags for today be unfeasible? I understand if it would be (these things can be super finnicky), but it could be nice to show.